addons/repair/functions/fnc_canRepair.sqf

```diff
@@ -35,10 +35,10 @@ private _engineerRequired = if (isNumber (_config >> "requiredEngineer")) then {
     };
     0;
 };
-if !([_caller, _engineerRequired] call FUNC(isEngineer)) exitWith {false};
+if !([_unit, _engineerRequired] call FUNC(isEngineer)) exitWith {false};
 
 private _items = _config call FUNC(getRepairItems);
-if (count _items > 0 && {!([_caller, _items] call FUNC(hasItems))}) exitWith {false};
+if (count _items > 0 && {!([_unit, _items] call FUNC(hasItems))}) exitWith {false};
```

Comment:
I think it'd be much simpler to have `hasItems` do `count _items` instead, less remembering stuff like this needed that way.